appveyor.yml

```diff
@@ -11,9 +11,9 @@ build_script:
 - ps: dotnet --info
 # Run dotnet
 - ps: dotnet restore
-- ps: dotnet build
-- ps: dotnet test
-- ps: dotnet pack -c Release --version-suffix=$revision
+- ps: dotnet build -c Release --no-restore
+- ps: dotnet test -c Release --no-build
```

Comment:
`-c Release` allows subsequent commands to use the outputs from previous commands.